soh/soh/OTRGlobals.h

```diff
@@ -12,8 +12,17 @@
 #define GAME_PLATFORM_N64 0
 #define GAME_PLATFORM_GC 1
 
-#define BTN_MODIFIER1 0x00040
-#define BTN_MODIFIER2 0x00080
+#define BTN_CUSTOM_MODIFIER1 0x0040
+#define BTN_CUSTOM_MODIFIER2 0x0080
+
+#define BTN_CUSTOM_OCARINA_NOTE_D4 ((CONTROLLERBUTTONS_T)0x00010000)
+#define BTN_CUSTOM_OCARINA_NOTE_F4 ((CONTROLLERBUTTONS_T)0x00020000)
+#define BTN_CUSTOM_OCARINA_NOTE_A4 ((CONTROLLERBUTTONS_T)0x00040000)
+#define BTN_CUSTOM_OCARINA_NOTE_B4 ((CONTROLLERBUTTONS_T)0x00080000)
+#define BTN_CUSTOM_OCARINA_NOTE_D5 ((CONTROLLERBUTTONS_T)0x00100000)
+#define BTN_CUSTOM_OCARINA_DISABLE_SONGS ((CONTROLLERBUTTONS_T)0x00200000)
+#define BTN_CUSTOM_OCARINA_PITCH_UP ((CONTROLLERBUTTONS_T)0x00400000)
+#define BTN_CUSTOM_OCARINA_PITCH_DOWN ((CONTROLLERBUTTONS_T)0x00800000)
```

Comment:
Considering these are replacing CVars for things like sharp, disable, etc, how do these work? Are they still customizable with the custom ocarina controls?

Reply:
Instead of being handled specially with cvars, they're handled as if they were buttons on the n64 controller that the game checks for like any other button. This also means that they've been put in the input editor as if they were any other button.